service/hub/server/config.go

```diff
@@ -82,6 +102,8 @@ func LoadConfig(fileName string) (*Config, error) {
 	if err = validate(c); err != nil {
 		return nil, fmt.Errorf("invalid config: %s", err)
 	}
+	// store client globally
+	K8sClient, _ = kubernetes.NewKubernetesClient(c.Server.Kubeconfig, "")
```

Comment:
Dupicated of line #58?

Reply:
L58 is a checker for kubeconfig, split checker is batter?